include/substrait/type/Type.h

```diff
@@ -417,7 +419,7 @@ class StringLiteral : public ParameterizedTypeBase {
   }
 
   [[nodiscard]] bool isWildcard() const override {
```

Comment:
This isn't how I would interpret the phrase "wildcard" (but that is probably a discussion for a separate issue).  I think `any` is a wildcard and `T` is a placeholder.  Maybe placeholder would be a better term.  But maybe I am wrong in my interpretation too.

Reply:
I added method  `isPlaceholder` to solve this.